### PR TITLE
Add HRM-9

### PR DESCRIPTION
**Pull request by paulhoenhorst**
Added all the HRM-9 (new SMG) stuff.

### CODE REVIEW

**Comment by paulhoenhorst** [comment]
@carlssonemil idk why vercel is failing here.

File: src/data/camouflages/skeletal.js [hunk 1]
```diff
@@ -20,6 +21,7 @@ const zombies = [
   'Thanatos',
   'Vehement Rage',
   'Harbinger'
```
Comment (carlssonemil) [inline src/data/camouflages/skeletal.js:23]: Missing a comma here, probably why the build fails on Vercel 😄 
```suggestion
  'Harbinger',
```

File: src/i18n/locales/en-US.json [hunk 2]
```diff
@@ -547,11 +547,13 @@
       "kills_while_ads_and_moving": "{amount} kills while aiming down sights and moving",
       "kills_while_ads_and_strafing": "{amount} kills while aiming down sights and strafing",
       "kills_while_ads": "{amount} kills while aiming down sights",
+      "kills_shortly_after_sprinting_ads":"get {amount} kills while aiming down sights shortly after sprinting",
```
Comment (carlssonemil) [inline src/i18n/locales/en-US.json:550]: No need for the "get" word in here 😄 
```suggestion
      "kills_shortly_after_sprinting_ads":"{amount} kills while aiming down sights shortly after sprinting",
```

**Review by carlssonemil — CHANGES_REQUESTED** [review]

**Comment by paulhoenhorst** [comment]
Thanks buddy! Commited the changes

File: src/data/requirements/camouflages/blur.js [hunk 3]
```diff
@@ -124,4 +124,22 @@ export default {
       type: 'kills_shortly_after_field_upgrade'
     }
   }
```
Comment (carlssonemil) [inline src/data/requirements/camouflages/blur.js:126]: Missing comma here too 😄 
```suggestion
  },
```

File: src/data/requirements/camouflages/blur.js [hunk 4]
```diff
@@ -124,4 +124,22 @@ export default {
...
+
+  'Ocean Waves': {
+    weapon: 'HRM-9',
+    level: '12',
+    challenge: {
+      amount: 100,
+      type: 'kills_while_crouching_or_sliding',
+    }
+  }
```
Comment (carlssonemil) [inline src/data/requirements/camouflages/blur.js:135]: And here
```suggestion
  },
```

**Review by carlssonemil — CHANGES_REQUESTED** [review]
Some more missing commas that I missed 😅 Let's see if these fixes the build issues!

**Pull request merged**